assemblies/pentaho-war/pom.xml

```diff
@@ -11,7 +11,6 @@
   <version>9.5.0.0-SNAPSHOT</version>
   <packaging>war</packaging>
   <properties>
-    <snakeyaml.version>1.13</snakeyaml.version>
```

Comment:
you can't delete this property without replace/updating it in other places. You are getting a build error now

https://wingman.orl.eng.hitachivantara.com/blue/organizations/jenkins/projects%2Fplatform-11/detail/PR-5182/5/pipeline


`${snakeyaml.version} `is used in:

- ./repository/pom.xml:190:      <version>${snakeyaml.version}</version>
- ./assemblies/pentaho-war/pom.xml:860:      <version>${snakeyaml.version}</version>
- ./extensions/pom.xml:1851:      <version>${snakeyaml.version}</version>